### ISSUE TITLE
Question: lein-cljsbuild connecting to 188.93.79.52:80

### ISSUE THREAD

**Issue by zoowar** [emezeske/lein-cljsbuild#412]
I'm using lein-cljsbuild-1.1.0. Today I it became extremely slow and I noticed it is trying to connect to 188.93.79.52:80 which is not responding and doesn't have a dns entry. Can someone shed some light on why lein-cljsbuild is connecting to 188.93.79.52:80?

**Comment by cemerick** [comment]
No idea on that one; cljsbuild itself doesn't touch the network at all. This could be caused by any Leiningen plugin…

**Comment by zoowar** [comment]
Thanks. I isolated this issue to lein-cljsbuild by disabling all plugins. When I removed lein-cljsbuild, the issue went away. However, with your comment in mind, I pushed harder and removed the archiva server from my :user profile, and the issue went away. It seems this issue requires both (not sure why lein-cljsbuild is involved). Anyway, I located the DNS entry for 188.93.79.52 which is clinker.netty.io. I have this configured as a remote archiva repo. It use to be the nightly snapshot server, and it looks like netty switched nightly snapshot server to https://oss.sonatype.org.
